expression/builtin_time_vec.go

```diff
@@ -1942,11 +1942,25 @@ func (b *builtinSubDateStringIntSig) vecEvalTime(input *chunk.Chunk, result *chu
 }
 
 func (b *builtinDateLiteralSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinDateLiteralSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	literal, isNull, err := b.evalTime(chunk.Row{})
```

Comment:
I think calling non-vec function in vec function is unreasonable.  We can use b.literal to calculate the result.

Reply:
Comment addressed. PTAL, Thanks. @wshwsh12